data/locale/en-US.ini

```diff
@@ -93,6 +101,7 @@ NVENC.Preset.Lossless="Lossless"
 NVENC.Preset.LosslessHighPerformance="Lossless High Performance"
 NVENC.RateControl="Rate Control Options"
 NVENC.RateControl.Mode="Mode"
+NVENC.RateControl.Mode.Description="Constant Quantization Parameter: A flat compression ratio with no regard for bit rates. This yields the highest quality-per-bitrate.\nVariable Bitrate: Where appropriate, sacrifices quality to stay below the upper bitrate limit, or saves bitrate where possible.\nHigh Quality Variable Bitrate: Variable Bitrate with two-pass encoding enabled by default.\nConstant Bitrate: Similar to Variable Bitrate, but performs bit stuffing to avoid falling below the target bitrate.\nHigh Quality Constant Bitrate: Constant Bitrate with two-pass encoding enabled by default.\nLow Delay High Quality Constant Bitrate: Optimized for lowest encoding latency, mainly by disabling B-frames and using slice multithreading."
```

Comment:
Please split the description here up into NVENC.RateControl.Mode.XXX.Description, as having one huge description breaks the layout on smaller monitors or high DPI monitors.